tests/switchless/enc/enc.c

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include "switchless_t.h"
+
+int standard_enc_sum(int arg1, int arg2)
+{
+    return arg1 + arg2;
+}
+
+int synchronous_switchless_enc_sum(int arg1, int arg2)
+{
+    return arg1 + arg2;
+}
+
+void batch_enc_sum(addition_args* args, size_t count)
+{
+    for (size_t i = 0; i < count; ++i)
+    {
+        args[i].sum = args[i].arg1 + args[i].arg2;
+    }
+}
+
+OE_SET_ENCLAVE_SGX(
+    1,    /* ProductID */
+    1,    /* SecurityVersion */
+    true, /* AllowDebug */
+    1024, /* HeapPageCount */
```

Comment:
We may want to reduce the page count to make the enclave run on Windows.

Reply:
@EMumau  Please reference below to see how @anitagov reduced page count for Windows 
https://github.com/openenclave/openenclave/commit/d6794b40b0229483ac8a800ea5b92ddea41fab5a

Reply:
Added to issue #2037